source/org/intellij/plugins/ceylon/ide/project/CeylonPageTwo.ceylon

```diff
@@ -300,10 +302,16 @@ shared class CeylonPageTwo()
     shared actual void load(IdeaCeylonProject config) {
         form.\imodule = config.ideArtifact;
         value systemRepository = config.ideConfiguration.systemRepository;
-        form.systemRepository.text = if (!exists systemRepository) then null else systemRepository.string;
+        //TODO: Investigate. `text` setter has annotation `@Nullable`,
```

Comment:
@FroMage Could you take a peek on it?
getter and setter signatures of `TextFieldWithBrowseButton` (`form.systemRepository`):
```
  @NotNull
  @Override
  public String getText();

  @Override
  public void setText(@Nullable String text);
```
However ceylon compiler is not accepting `form.systemRepository.text = if (!exists systemRepository) then null else systemRepository.string;` complaining that you can't assign `String|Null` to `String`.